packages/shared/providers.yaml

```diff
@@ -4009,6 +4085,17 @@ lever-basic-sandbox:
     proxy:
         base_url: https://api.sandbox.lever.co
     docs: https://docs.nango.dev/integrations/all/lever
+    credentials:
+        username:
+            type: string
+            title: User name
+            description: The API Key of your lever sandbox account
```

Comment:
```suggestion
            description: The API Key of your Lever sandbox account
```

Reply:
Thanks Thomas, updated with the latest [commit](https://github.com/NangoHQ/nango/pull/3190/commits/1b0d7e01c8c4395b3d0f687cbce67e3932849877)